tests/generation/test_configuration_utils.py

```diff
@@ -204,8 +204,9 @@ def test_validate(self):
 
         # By default we throw a short warning. However, we log with INFO level the details.
         # Default: we don't log the incorrect input values, only a short summary. We explain how to get more details.
-        with CaptureLogger(logger) as captured_logs:
-            GenerationConfig(do_sample=False, temperature=0.5)
+        with LoggingLevel(logging.WARNING):
```

Comment:
there is something in our test suite that likely changes the default logging level -- this test is flaky because the line below is capturing INFO level logs 🤔 

these flaky checks were merged today by me (2025-may-19)